### ISSUE TITLE
Managing the external resources (uri paths) from gltf extensions

### ISSUE THREAD

**Issue by fire** [omigroup/gltf-extensions#66]
Propose an OMI extension for external resources.

<img src="https://user-images.githubusercontent.com/32321/154621682-52d8ff40-c34f-4a82-a1cf-3fdbb20bf212.png" width="500">

* gltf have external resources that may cause security risks.
* Resources may become significantly more fragile due the passage of time with the links breaking.
* Privacy may leak from access.

Process for addressing:

1. gltf transform workflow
2. Consent for playback and tracking
3. Static bundles
4. Self-contained bundles
5. Link checking tool
6. List exposed providers

## Levels

1. Whitelist all external resources
1. Block all external resource
1. All access external resources

## Prior work

* https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity
* https://datatracker.ietf.org/doc/html/rfc2577

**Comment by oveddan** [comment]
yes this would be great! Also I would add it would be great if this supported [IPFS](https://ipfs.io/) addresses, as that is what's often used for storing files linked to the blockchain - and with `IPFS` supported here you could have assets with globally unique permanent addresses that you could host yourself by pinning the content.

**Comment by oveddan** [comment]
Also it would be nice if it supported relative paths, that way files on the same host/domain can reference each other relatively, and you don't have to worry about a changing domain breaking things.

**Comment by npolys** [comment]
See ISO X3D for a Web-ready semantic (fallbacks etc):
https://web3d.org/documents/specifications/19775-1/V4.0/Part01/components/networking.html
